server/etcdserver/raft.go

```diff
@@ -314,6 +327,41 @@ func (r *raftNode) start(rh *raftReadyHandler) {
 	}()
 }
 
+// For a cluster with only one member, the raft always send identical
```

Comment:
Note about comments, it should be written from a point for the issue being fixed. Issue that is fixed can be just linked.

Reply:
Updated & thx